pulsar-flink-connector/pom.xml

```diff
@@ -84,6 +84,11 @@ under the License.
 			<version>${flink.version}</version>
 			<scope>provided</scope>
 		</dependency>
+		<dependency>
+			<groupId>javax.xml.bind</groupId>
+			<artifactId>jaxb-api</artifactId>
+			<version>${jaxb.version}</version>
+		</dependency>
```

Comment:
Why we need this?

Reply:
The JAXB APIs are considered to be Java EE APIs and therefore are no longer contained on the default classpath in Java SE 9. In Java 11, they are completely removed from the JDK.

Without this dependency, maven will report class not found issue for: https://github.com/streamnative/pulsar-flink/blob/master/pulsar-flink-connector/src/main/java/org/apache/flink/streaming/connectors/pulsar/internal/DateTimeUtils.java#L23

Reply:
DatatypeConverter could be dropped. I don't like one method dependency jar.